js/reducers/report-reducer.ts

```diff
@@ -121,7 +124,7 @@ export default function report(state = new ReportState({}), action?: any) {
   switch (action.type) {
     case RECEIVE_PORTAL_DATA:
       data = preprocessPortalDataJSON(action.response);
-
+      
```

Comment:
Note that this now throws a lint warning:
```
D:\Repos\portal-report\js\reducers\report-reducer.ts
  127:1  warning  Trailing spaces not allowed  no-trailing-spaces
```
We'll want to fix this.  Might be best to change your VS Code settings to strip whitespace.

Reply:
Fixed